olive/systems/azureml/aml_system.py

```diff
@@ -276,6 +277,12 @@ def _create_model_args(self, model_json: dict, model_resource_paths: Dict[str, R
 
         return args
 
+    def _create_olive_config_file(self, olive_config: dict, tmp_dir: Path):
+        olive_config_path = tmp_dir / "olive_config.json"
+        with olive_config_path.open("w") as f:
```

Comment:
need check whether the olive_config is None

Reply:
Ideally olive_config should not be None for AML system. We want config for all aml jobs. But if there is any error it is None, this config json file will be null.

Reply:
The UT or integration test might be null

Reply:
If so, null would be expected right? What kind of check do you suggest here?

Reply:
If the olive_config is None, we could skip append the file to copy_files at line 375? so only cur_dir / script_name will be copied to code path?

Reply:
Updated.